.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Node
+        id: setup-node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Build Node artifacts
+        run: yarn install && yarn build
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [16, 18]
```

Comment:
I think we'll also want to add `12` and `14`, given that we specify Node 12+ is supported in the readme.